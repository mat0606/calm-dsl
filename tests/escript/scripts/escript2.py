# python2;success

print uuid.uuid4() == uuid.uuid4()

val = """number: 3.14
string: !!str 3.14
"""
pprint(yaml.safe_load(val))
