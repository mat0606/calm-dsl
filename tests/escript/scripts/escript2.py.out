False
{'number': 3.14, 'string': '3.14'}
